### ISSUE TITLE
Game files are no longer deleted

### ISSUE THREAD

**Issue by PeterPiers** [ifrotz/iosfrotz#261]
Hey, I did notice something was weird after the newest iOS update, but I may now have figured out what.

Previously, I could access many apps with DiskAid (an explorer program for the iOS). As of iOS 8.3, though, only apps with sharing enabled (or whatever it is) can be accessed this way.

Frotz, however, was an app that I could always access even though it didn't seem to HAVE that sharing funcionality enabled. I shrugged and moved on.

Later, when updating games, it seemed to me as though Frotz was failing to delete game files... and I've now discovered I can't delete them myself either (using iMazing to go into "\Games\", I can add files to that directory but not remove them).

Basically, I think iFrotz is choking somewhere with some changes brought on by 8.3. It doesn't seem to be deleting a game file when it should. It doesn't seem to even allow that to happen.

If this is too confusing to follow let me know and I'll try to explain better.

EDIT - After having read up on your difficulties with the iTunes file sharing, I wonder whether this problem will have an easy solution... am thinking it might not.

**Comment by PeterPiers** [comment]
Never mind this one. Further proper testing showed this is not the case. Urgh. I still haven't learned to test PROPERLY and THOROUGHLY before I jump the gun.

**Comment by PeterPiers** [comment]
Oh, I can close things? Whoops. Didn't know.
